Bot/Localize.py

```diff
@@ -19,13 +19,13 @@
 }
 
 CancelChangingGit = {
-    'en': "Сancelled changing.🔙",
-    'ru': "Изменение отменено.🔙",
+    'en': "Сancelled changing 🔙",
+    'ru': "Изменение отменено 🔙",
 }
 
 Rules = {
     'en': "Every day at 12:00 UTC-0 (London) we send you 5 GitHub users to follow. If you follow them in 24 hours, then you will be featured to other users too. If you miss it, that's okay. As soon as you follow them we will add you back to the featured list.",
-    'ru': "Каждые сутки в 12 UTC-0 (London) мы присылаем 5 пользователей. Если ты зафолловишь всех, то попадаешь в следующую рассылку.\nЕсли пропустишь - ничего страшного, подпишешься, и мы добавим тебя обратно в рассылку.",
+    'ru': "Каждые сутки в 12 UTC-0 (London) мы присылаем 5 пользователей. Если ты подпишешься на всех, то попадаешь в следующую рассылку.\nЕсли пропустишь - ничего страшного, подпишешься, и мы добавим тебя обратно в рассылку.",
```

Comment:
Given that UK has daylight saving time, London is not in UTC+0 only, but either in GMT/GMT+1 time zone. Let's just use `UTC` and remove `London` in this message.